gen/dcos-config.yaml

```diff
@@ -367,22 +367,31 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
-  - path: /etc/mesos-slave-dns.json 
+  - path: /etc/mesos-slave-dns.json
     content: |
       {
         "docker":
         [
             {
               "network_mode": "USER",
               "dns": {
-                "nameservers": [ 
+                "nameservers": [
                                 "198.51.100.1",
                                 "fd01:d::c633:6401"
                 ]
               }
             }
         ]
       }
+  - path: /etc/mesos-slave-image-gc-config.json
```

Comment:
s/slave/agent/g ?

Reply:
Rebased. Thanks, @adam-mesos !

About the renaming, we may not want to do that now. Because we may need to name all other agent related config files as `mesos-agent-xxxxx`, for instance, now we have:
/opt/mesosphere/etc/mesos-slave
/opt/mesosphere/etc/mesos-slave-public
/opt/mesosphere/etc/mesos-slave-dns.json
/opt/mesosphere/etc/mesos-slave-common
/opt/mesosphere/etc/mesos-slave-modules
/opt/mesosphere/etc/mesos-slave-public